lib/new_relic/agent/configuration/default_source.rb

```diff
@@ -1595,6 +1595,14 @@ def self.enforce_fallback(allowed_values: nil, fallback: nil)
           :allowed_from_server => false,
           :description => 'Controls auto-instrumentation of Sinatra at start up. May be one of: `auto`, `prepend`, `chain`, `disabled`.'
         },
+        :'instrumentation.stripe' => {
+          :default => 'enabled',
+          :documentation_default => 'enabled',
```

Comment:
I don't think you have to add `documentation_default` when the `default` value is not dynamic (ex. uses a method or a proc to resolve the value)

Reply:
Removed! https://github.com/newrelic/newrelic-ruby-agent/pull/2180/commits/d577fddc85e8d4f31c64ec12e128fa6e4ab2f5f9